### PR TITLE
Add Fuel Detotalizer Switch to the Mirage 2000C Fuel Panel

### PR DESCRIPTION
**Pull request by norsetto**
Hi again,

I added the Fuel Detotalizer switch to the Mirage 2000C Fuel Panel. Unfortunately I couldn't figure out a way to add the bindings if not manually via the Profile Editor. Could this be of interest to you, and if so, could you perhaps advise me on how to add the bindings directly?

### CODE REVIEW

**Comment by derammo** [comment]
what you have there should theoretically work.  However, the "CompositeVisual" stuff is a horrible mess where people repeatedly copied the same code to a bunch of classes, so it is very likely broken somewhere.   You will need to step through the code to see why it doesn't populate the default bindings correctly.   There are other, very similar switches in there you could verify first to make sure stuff is working.   If you can't get it worked out, let me know and I will spin up on this.

**Comment by norsetto** [comment]
I suspect that this is because the declutter switch is a momentary switch.

**Comment by derammo** [comment]
@norsetto I can't find your Discord info, can you friend me on Discord please?

File: Helios/Gauges/M2000C/PCNPanel/PCN_Panel.cs [hunk 1]
```diff
@@ -49,7 +49,7 @@ public M2000C_PCNPanel()
             AddIndicatorPushButton("EFF", "eff", new Point(column5, row8), new Size(50, 50));
             AddIndicatorPushButton("INS", "ins", new Point(column7, row8), new Size(50, 50));
 
-            AddIndicatorPushButton("Offset Waypoint/Target", "enc", new Point(column1, row1), new Size(58, 40));
+            //AddIndicatorPushButton("Offset Waypoint/Target", "enc", new Point(column1, row1), new Size(58, 40));
```
Comment (derammo) [inline Helios/Gauges/M2000C/PCNPanel/PCN_Panel.cs:52]: please add a comment why this is removed to clarify to the next reader

File: Helios/Interfaces/DCS/M2000C/M2000CInterface.cs [hunk 2]
```diff
@@ -407,8 +407,8 @@ public M2000CInterface()
             AddFunction(new FlagValue(this, "571", "PCN Panel", "PREP", "PREP"));
             AddFunction(new PushButton(this, PCN_NAV, "3572", "572", "PCN Panel", "DEST Button"));
             AddFunction(new FlagValue(this, "573", "PCN Panel", "DEST", "DEST"));
-            AddFunction(new PushButton(this, PCN_NAV, "3667", "667", "PCN Panel", "Offset Waypoint/Target Button"));
-            AddFunction(new FlagValue(this, "668", "PCN Panel", "Offset Waypoint/Target", "ENC"));
+            //AddFunction(new PushButton(this, PCN_NAV, "3667", "667", "PCN Panel", "Offset Waypoint/Target Button"));
```
Comment (derammo) [inline Helios/Interfaces/DCS/M2000C/M2000CInterface.cs:410]: please add a comment why this is removed

File: Helios/Interfaces/DCS/M2000C/M2000CInterface.cs [hunk 3]
```diff
@@ -407,8 +407,8 @@ public M2000CInterface()
...
+            //AddFunction(new FlagValue(this, "668", "PCN Panel", "Offset Waypoint/Target", "ENC"));
```
Comment (derammo) [inline Helios/Interfaces/DCS/M2000C/M2000CInterface.cs:411]: please add a comment why this is removed

**Review by derammo — CHANGES_REQUESTED** [review]

**Comment by norsetto** [comment]
No problem, I added comments as well as removed two unused variables which were left from the change.

**Review by derammo — APPROVED** [review]

**Pull request merged**